haiku-games/bebattle/bebattle-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="The BeBattle board game is a fun strategy game"
+DESCRIPTION="The BeBattle game is a fun board strategy game."
+HOMEPAGE="https://github.com/HaikuArchives/BeBattle/"
+COPYRIGHT="1999-2000 Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+COMMIT="e44b8fd01f895e1ef068ab330abd3e492d95d369"
+SOURCE_URI="https://github.com/HaikuArchives/BeBattle/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="30be929f3fb9725fc8ca6f817fb75abd45c526436154d5dc397f39940b48f583"
+SOURCE_DIR="BeBattle-$COMMIT"
+
+ARCHITECTURES="x86_gcc2 !x86"
+
```

Comment:
Did you check if it builds with gcc5?

Reply:
Yes, and it didn't.

Reply:
Maybe you could add "!x86_64" there too then :)

Reply:
Maybe a question mark? I can't test 64-bit as I am running in 32-bit VM.

Reply:
x86_64 is pure gcc5, if it doesn't build for x86 then probably not for x86_64